release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3354,6 +3354,57 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, the cluster autoscaler stopped scaling because a machine failed in a machine set. This situation happened because of inaccuracies in the way the cluster autoscaler counts machines in various non-running phases. With this release, the inaccuracies have been fixed so that the cluster autoscaler has a more accurate count. (link:https://issues.redhat.com/browse/OCPBUGS-54325[OCPBUGS-54325]) 
+
+ * Previously, an update to the {ibm-cloud-name} Cloud Internet Services (CIS) implementation impacted the upstream Terraform plugin. If there was an attempt to create an external-facing cluster on {ibm-cloud-name}, the following error occurred:
++
+[source,terminal]
+----
+ERROR Error: Plugin did not respond
+ERROR
+ERROR with module.cis.ibm_cis_dns_record.kubernetes_api_internal[0],
+ERROR on cis/main.tf line 27, in resource "ibm_cis_dns_record" "kubernetes_api_internal":
+ERROR 27: resource "ibm_cis_dns_record" "kubernetes_api_internal"
+----
++
+With this release, the installation program can be used to create an external cluster on {product-title} without the plugin issue. (link:https://issues.redhat.com/browse/OCPBUGS-54367[OCPBUGS-54367]) 
+
+
+* Previously, the Operator Marketplace and the {olm-first} used an older version of the `pod-security.kubernetes.io/` label. With this release, the namespace where the Operator Marketplace is deployed now uses the Pod Security Admission (PSA) label marked as `latest`. (link:https://issues.redhat.com/browse/OCPBUGS-53283[OCPBUGS-53283])
+
+* Previously, a User Datagram Protocol (UDP) packet that was larger than the maximum transmission unit (MTU) value set for the cluster could not be sent to the endpoint of the packet by using a service. With this release, the pod IP address is used instead of the service IP address regardless of the packet size, so that the UDP packet can be sent to the endpoint. (link:https://issues.redhat.com/browse/OCPBUGS-50581[OCPBUGS-50581]) 
+
+* Previously, when installing a cluster on {aws-first} in existing subnets that were located in edge zones, such as a Local Zone or a Wavelength Zone, the `kubernetes.io/cluster/<InfraID>:shared` tag was missing in the subnet resources of the edge zone. With this release, a fix ensures that all subnets that are used in the `install-config.yaml` configuration file have the required tag. (link:https://issues.redhat.com/browse/OCPBUGS-49975[OCPBUGS-49975]) 
+
+* Previously, if a build that required a trust bundle to access registries was run, the build did not pick up the bundle configured in the cluster proxy. The builds failed if a registry required for a custom trust bundle was referenced. With this release, builds that require the trust bundle specified in the proxy configuration succeed, and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-45268[OCPBUGS-45268]) 
+
+
```

Comment:
Add:

```
[id="ocp-4-16-39-updating_{context}"]
==== Updating

To update an existing {product-title} {product-version} cluster to this latest release, see xref:../updating/updating_a_cluster/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```

Reply:
Consider contacting Roman Dobosz about https://issues.redhat.com/browse/OCPBUGS-53248. Although it's marked as internal (Red Hat Employee only), they might have made a mistake. They just might need to remove customer info before they change the status.